config/yamlcfg/yaml2proto_test.go

```diff
@@ -256,6 +256,33 @@ test_groups:
 - name: testgroup_1
 `),
 		},
+		{
```

Comment:
One more case for rejecting empty column headers as well?

Reply:
Done